test/e2e/serial/tests/configuration.go

```diff
@@ -446,10 +427,12 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
 			}).WithTimeout(10 * time.Minute).WithPolling(30 * time.Second).Should(Succeed())
 
 			By("waiting for mcp to start updating")
-			waitForMcpsCondition(fxt.Client, context.TODO(), mcps, machineconfigv1.MachineConfigPoolUpdating)
+			err = waitForMcpsCondition(fxt.Client, context.TODO(), mcps, machineconfigv1.MachineConfigPoolUpdating)
+			Expect(err).ToNot(HaveOccurred())
 
 			By("wait for mcp to get updated")
-			waitForMcpsCondition(fxt.Client, context.TODO(), mcps, machineconfigv1.MachineConfigPoolUpdated)
+			err = waitForMcpsCondition(fxt.Client, context.TODO(), mcps, machineconfigv1.MachineConfigPoolUpdated)
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
Here you can also use `waitForMcpUpdate` and some other places below

Reply:
indeed I'll update these, thanks for catching